user_docs/en/userGuide.t2t

```diff
@@ -107,10 +107,10 @@ There are also a few options in this dialog which are explained below.
 Once the installation has completed, a message will appear telling you that it was successful.
 Pressing OK at this point will restart the newly installed copy of NVDA.
 
-+++ Untested add-ons warning +++[InstallWithUntestedAddons]
-If you have add-ons already installed there may also be a warning that untested add-ons will be disabled.
++++ Incompatible add-ons warning +++[InstallWithIncompatibleAddons]
+If you have add-ons already installed there may also be a warning that incompatible add-ons will be disabled.
 Before you're able to press the Continue button you will have to use a checkbox to confirm that you understand that these add-ons will be disabled.

```

Comment:
Hmm, may be change use a checkbox into check a checkbox?

Reply:
I think check a checkbox sounds a little strange. It probably should be "use the checkbox".

Reply:
Using a check box sounds a bit neutral to me. Either checking or unchecking it could be treated as usage of it.

How about enable the check box?